src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -56,70 +61,160 @@ public static AbstractResponse create(String appName, String appVersion, int jre
 		return res;
 	}
 
+    /**
+     * Gets the app name
+     *
+     * @return The app name.
+     */
 	public String getAppName() {
 		return appName;
 	}
-
+    
+    /**
+     * Name of the IOTA software you're currently using (IRI stands for Initial Reference Implementation)
+     *
+     * @return The app version.
+     */
 	public String getAppVersion() {
 		return appVersion;
 	}
 
+    /**
+     * Available cores on your machine for JRE.
+     *
+     * @return The available processors of the JRE.
+     */
 	public int getJreAvailableProcessors() {
 		return jreAvailableProcessors;
 	}
 
+    /**
+     * The amount of free memory in the Java Virtual Machine.
+     *
+     * @return The free memory of the JRE.
+     */
 	public long getJreFreeMemory() {
 		return jreFreeMemory;
 	}
 
+    /**
+     * The maximum amount of memory that the Java virtual machine will attempt to use.
+     *
+     * @return The maximum memory of the JRE.
+     */
 	public long getJreMaxMemory() {
 		return jreMaxMemory;
 	}
 
+    /**
+     * The total amount of memory in the Java virtual machine.
+     *
+     * @return The total memory of the JRE.
+     */
 	public long getJreTotalMemory() {
 		return jreTotalMemory;
 	}
 
+    /**
+     * The JRE version this node runs on
+     *
+     * @return The JRE version.
+     */
 	public String getJreVersion() {
 		return jreVersion;
 	}
 
+    /**
+     * The hash of the latest transaction that was signed off by the coordinator.
+     *
+     * @return The latest milestone.
+     */
 	public String getLatestMilestone() {
 		return latestMilestone;
 	}
 
+    /**
+     * Index of the latest milestone.
+     *
+     * @return The latest milestone index.
+     */
 	public int getLatestMilestoneIndex() {
 		return latestMilestoneIndex;
 	}
 
+    /**
+     * The hash of the latest transaction which is solid and is used for sending transactions. 
+     * For a milestone to become solid your local node must basically approve the subtangle of coordinator-approved transactions, 
+     *  and have a consistent view of all referenced transactions.
+     *
+     * @return The latest subtangle milestone index.
+     */
 	public String getLatestSolidSubtangleMilestone() {
 		return latestSolidSubtangleMilestone;
 	}
 
+    /**
+     * Index of the latest solid subtangle.
+     * 
+     * @return The latest subtangle milestone index.
+     */
 	public int getLatestSolidSubtangleMilestoneIndex() {
 		return latestSolidSubtangleMilestoneIndex;
 	}
 
+    /**
+     * Gets the start of the milestone index
+     *
+     * @return The start of the milestone index.
```

Comment:
English nit:
"start of" sounds strange...
`@return the milestone start index` maybe?

Or just remove this javadoc entirely if it not needed for md.